extensions/redis-cache/runtime/src/main/java/io/quarkus/cache/redis/runtime/RedisCacheRuntimeConfig.java

```diff
@@ -9,11 +9,28 @@
 @ConfigGroup
 public class RedisCacheRuntimeConfig {
     /**
-     * The default time to live of the item stored in the cache
+     * The default time to live of the item stored in the cache.
+     *
+     * @deprecated Use {@link #expireAfterWrite} instead.
      */
     @ConfigItem
+    @Deprecated
```

Comment:
Let's please add a note that users should `expireAfterWrite` instead

Reply:
It's in the javadoc. I don't think I can add something in the annotation.

Reply:
Ah sorry, I didn't see it was already there, my bad